dev-tools/commands/src/purchase-track.mjs

```diff
@@ -1,38 +1,47 @@
-import chalk from "chalk";
-import { program } from "commander";
+import chalk from 'chalk'
```

Comment:
Both of the `.mjs` files got reformatted on save, following the prettier config that already existed in the repo.

Reply:
Oh good call. I can do that in the future. I also think it's worth just doing a formatting pass where we run prettier on the repo.